services/github/github-tag.spec.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const { test, given } = require('sazerac')
+const { addv } = require('../text-formatters')
+const { version: versionColor } = require('../color-formatters')
+const GithubTag = require('./github-tag.service')
+
+describe('GithubTag', function() {
+  const tagFixture = [
+    { name: 'cheese' }, // any old string
+    { name: 'v1.3-beta3' }, // semver pre-release
+    { name: 'v1.2' }, // semver release
+  ]
+
+  test(GithubTag.transform, () => {
+    given({ json: tagFixture, usingSemver: true, includePre: false }).expect(
+      'v1.2'
+    )
+    given({ json: tagFixture, usingSemver: true, includePre: true }).expect(
+      'v1.3-beta3'
+    )
+    given({ json: tagFixture, usingSemver: false, includePre: false }).expect(
+      'cheese'
+    )
+  })
+
+  test(GithubTag.render, () => {
+    given({ usingSemver: false, version: '1.2.3' }).expect({
+      message: addv('1.2.3'),
```

Comment:
Any reason not to replace these with literals?

Reply:
I've no issues with replacing them. Rationale very much related to https://github.com/badges/shields/pull/3266#discussion_r278374394, the latter paragraph in particular

Reply:
`message`'s inlined, still calling the `versionColor` helper below but happy to change that to `blue` if you prefer (and re 👍 the PR)

Reply:
I'm happy to continue the discussion at https://github.com/badges/shields/pull/3266#discussion_r278374394 though that's a bit more long-ranging; we can revisit this. For now I'd prefer we inline the color as well.